lib/modules/datasource/github-releases/types.ts

```diff
@@ -13,6 +13,7 @@ export type GithubRelease = {
 
 export interface GithubReleaseAsset {
   name: string;
+  url: string;
   browser_download_url: string;
```

Comment:
We probably need to use `url` in other datasources too? otherwise it will fail for private repos as @lyonlai  suggested?

Reply:
@HonkingGoose please file an issue for this 🤗

Reply:
Done, see:

- #16575